modules/installation-network-user-infra.adoc

```diff
@@ -114,24 +114,34 @@ for platform containers and provide telemetry data to Red Hat.
 ====
 endif::[]
 
-.Load balancers
-
-Before you install {product-title}, you must provision two layer-4 load
-balancers. The API requires one load balancer and the default Ingress Controller
-needs the second load balancer to provide ingress to applications.
-
+[id="installation-network-user-infra-load-balancers_{context}"]
+=== Load balancers
+
+Before you install {product-title}, you must provision two load balancers that meet the following requirements:
+
+. *API Load Balancer*: Provides a common endpoint for users, both human and machine, to interact with and configure the platform. Configure one of the following strategies:
++
+--
+  ** Layer 4 load balancing only. This is sometimes referred to as Raw TCP, SSL Passthrough, or SSL Bridge mode.
+  ** Round robin load balancing.
+--
++
+Configure the following ports on both the front and back of the load balancers:
++
+.API load balancer
 [cols="2a,5a,2a,2a,2a",options="header"]
 |===
 
 |Port
-|Machines
+|Back-end machines (pool members)
 |Internal
 |External
 |Description
 
 |`6443`
 |Bootstrap and control plane. You remove the bootstrap machine from the load
-balancer after the bootstrap machine initializes the cluster control plane.
+balancer after the bootstrap machine initializes the cluster control plane. You
+must configure the `/readyz` endpoint for the API server health check probe.
```

Comment:
why not be clear about the timings?
```
You must configure the API server `/readyz` endpoint as the health check of the LB: within at most 30s after `/readyz` returns error or becomes healthy, the endpoint must be removed or added. Probing every 5 or 10 seconds, with 2 successful requests to become healthy and 3 to become unhealthy, are well-tested values.
```

Reply:
@sttts I added more detail for timings in 5ec28c182871d4ecbff344c68daf7d9718867510. Can you verify?

Reply:
Commented. 30s is really an upper bound. That's why I wrote "at most".

Reply:
Since the first sentence in this note states the 30 second max upper bound, I generically refer to the time frame afterwards since it can vary.

Reply:
looks good.